content/docs/join_us/recruitment_FAQs.md

```diff
@@ -52,11 +72,16 @@ In general, we suggest to wait for at least one year before applying again.
 
 ### Does the Turing cover relocation and visa costs?
 
-Yes.
-The Turing recognises that relocating and applying for a visa have different associated costs and therefore will provide support for both.
+Yes; the Turing recognises that relocating and applying for a visa have different associated costs and therefore will provide support for both.
+The Turing will reimburse up to:
+
+- £4000 for the cost of a visa application (including the NHS immigration surcharge);
+- £2000 for relocation costs (if you are moving from Europe or the UK, as long as the move is over 50 miles); and
+- £4000 for relocation costs (if you are moving from further overseas).
+
 All claims should be submitted within a year of the employment start date to Finance.
-For existing staff, visa claims should be submitted within one year of the visa renewal or amendment.
-More information can be found on [Mathison](https://mathison.turing.ac.uk/Interact/Pages/Section/ContentListing.aspx?subsection=4141&utm_source=interact&utm_medium=quick_search&utm_term=relocation).
+For existing staff, visa claims should be submitted within one year of the visa renewal or amendment: more information can be found on [Mathison](https://mathison.turing.ac.uk/Interact/Pages/Section/ContentListing.aspx?subsection=4141&utm_source=interact&utm_medium=quick_search&utm_term=relocation).
```

Comment:
Probably need to add a note that I think people can only access Mathison once they join the Turing

Reply:
I removed this as it's not really relevant for people applying to the job, you only need to know this once you start (and hopefully we'll have a proper handbook page on it in the near future).